### ISSUE TITLE
Not finding libastrochem.so.0

### ISSUE THREAD

**Issue by BhaswatiMookerjea** [smaret/astrochem#47]
I downloaded the latest astrochem from github. For the configure all parameters except the openMP are alright. After doing a `sudo make install` when I try to run astrochem I get the following error:

```
astrochem: error while loading shared libraries: libastrochem.so.0: cannot open shared object file: No such file or directory
```

Although the `/usr/local/lib` shows the following:

```
lrwxrwxrwx 1 root root     21 Feb 12 21:08 /usr/local/lib/libastrochem.so -> libastrochem.so.0.0.0
lrwxrwxrwx 1 root root     21 Feb 12 21:08 /usr/local/lib/libastrochem.so.0 -> libastrochem.so.0.0.0
-rwxr-xr-x 1 root root 109286 Feb 12 21:08 /usr/local/lib/libastrochem.so.0.0.0
```

This problem could be bypassed by copying these files to `/usr/lib` by hand.

**Comment by smaret** [comment]
Thanks for the report. 

What version of Astrochem do you use? On which platform?

**Comment by BhaswatiMookerjea** [comment]
It's 0.7-beta

**Comment by smaret** [comment]
Do you still get the error with `v0.7`?

**Comment by smaret** [comment]
I've bumped into the same error while trying to install `astrochem` on Ubuntu 16.04. It seems that `/usr/local/lib` isn't in Ubuntu's default path. You can fix this by setting the `LD_LIBRARY_PATH` environment path or with `ldconfig`. I'll add some instructions on the wiki page soon.

**Comment by smaret** [comment]
I've been thinking over this, and ideally one should not need to set the `LD_LIBRARY_PATH` environnement variable at run time. This can probably fixed with some `libtool` during the compilation.

**Comment by smaret** [comment]
I think that this should be fixed with 50bd5c089f973031c02b7331fd56a8972a157ca6. Closing this, but please re-open if you still get the issue.
